frontend/src/pages/modelCatalog/screens/ModelDetailsPage.tsx

```diff
@@ -103,12 +103,12 @@ const ModelDetailsPage: React.FC = conditionalArea(
       data-testid="tune-model-popover"
       minWidth="min-content"
       aria-label="Popover for fine tuning the model"
-      headerContent="How to tune this model?"
+      headerContent="LAB-tune this model?"
       headerIcon={<RhUiControlsIcon />}
       bodyContent={
         <div>
-          To fine-tune this model, you must first register it to an OpenShift AI model registry,
-          then click Lab tune.
+          To LAB-tune this model, you must first register it as a version to an OpenShift AI model
```

Comment:
As noted earlier - good catch @manaswinidas , we should use `ODH_PRODUCT_NAME` here.

Reply:
Changed to `ODH_PRODUCT_NAME`.

<img width="392" alt="Screenshot 2025-03-13 at 11 59 27 AM" src="https://github.com/user-attachments/assets/0c462fd2-4b6a-4cb2-970a-d3ac80c932ad" />